### DISCUSSION TITLE
ffsubsync on Linux

### DISCUSSION THREAD

**Discussion by SWBZA** [smacke/subsync#142] [Q&A] [answered]
Hi.

I don't know where to ask this, so I'm trying here.  Is there a version that runs on Linux, or a way to compile it for Linux?

Kind regards
Schalk

**Comment by smacke** [comment, marked as answer]
Hi there! It should be cross-platform and run fine on linux -- the only difference is that you'll need to figure out how to grab ffmpeg for your platform. If ubuntu, it's probably `sudo apt-get install ffmpeg`. After that it's just `python -m pip install ffsubsync` and everything should be working, provided that the ffmpeg executable is somewhere on your `$PATH` environment variable. Please let me know if that works; I'll keep this issue open for a few days or until I hear that it does :)

**Reply by SWBZA** [reply]
Hi.  

It looks like ffsubsync installed, but I'm having issues installing ffmpeg.  You're right, from my reading is should actually be installed already (I'm running Debian), but I get unmet dependencies error - see below.  I might try again later this week to install ffmpeg.  If I get it right, I'll test ffs and let you know the result.

Kind regards

```
sudo apt install ffmpeg -y                                                                                                                                    
[sudo] password for schalk: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 ffmpeg : Depends: libavcodec58 (= 7:4.3.3-0+deb11u1)
          Depends: libavdevice58 (= 7:4.3.3-0+deb11u1) but it is not going to be installed
          Depends: libavfilter7 (= 7:4.3.3-0+deb11u1)
          Depends: libavformat58 (= 7:4.3.3-0+deb11u1) but 7:4.4-6+b2 is to be installed
          Depends: libavresample4 (= 7:4.3.3-0+deb11u1) but it is not going to be installed
          Depends: libavutil56 (= 7:4.3.3-0+deb11u1) but 7:4.4-6+b2 is to be installed
          Depends: libpostproc55 (= 7:4.3.3-0+deb11u1) but 7:4.4-6+b2 is to be installed
          Depends: libswresample3 (= 7:4.3.3-0+deb11u1) but 7:4.4-6+b2 is to be installed
          Depends: libswscale5 (= 7:4.3.3-0+deb11u1) but 7:4.4-6+b2 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by smacke** [comment]
Hmm, this is probably out of scope for an ffsubsync issue I'm afraid. Maybe try searching the ask ubuntu stackexchange? I did a search there and this looks like it could be relevant: https://askubuntu.com/questions/110656/ffmpeg-unmet-dependencies
